.github/dependabot.yml

```diff
@@ -41,3 +41,25 @@ updates:
       - "ok-to-test"
       - "kind/cleanup"
       - "release-note-none"
+
+  # Backstage
+  - directory: "/backstage"
+    package-ecosystem: "npm"
+    open-pull-requests-limit: 5
+    schedule:
+      interval: "weekly"
+      time: "09:00"
+      # Use America/New_York Standard Time (UTC -05:00)
+      timezone: "America/New_York"
+    groups:
+      all-backstage-patch-updates:
+        patterns: [ "*" ]
+        update-types:
+          - "patch"
```

Comment:
Do we want to specify this ?
Does this mean only create updates for `patch` updates ? If so, so minor and major need to be updated manually ?

Reply:
yeah, I was thinking because we can't be sure if backstage will work with minor version update combinations, but we can be more confident for only patch version updates